static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -120,6 +124,8 @@ define([
 
         return applyCreativeTemplate(advert.node).then(function (isRendered) {
             return renderAdvertLabel(advert.node)
+                .then(addFeedbackDropdownToggle())
+                .then(applyFeedbackOnClickListeners(slotRenderEvent.creativeId))
```

Comment:
You will have to wrap that call in a lambda, otherwise the operation will be executed synchronously:
```
.then(function () { return applyFeedbackOnClickListeners(slotRenderEvent.creativeId); })
```